src/box/lua/schema.lua

```diff
@@ -84,6 +84,9 @@ ffi.cdef[[
     box_txn_savepoint_t *
     box_txn_savepoint();
 
+    int
+    box_advance_max_id(void);
```

Comment:
This doesn't match the actual function signature.

Reply:
Removed it (use LuaC now)